### DISCUSSION TITLE
Reopen locked month

### DISCUSSION THREAD

**Discussion by Slomo17** [kimai/Paid-plugins#172] [General]
Hello, i wanted to ask, if it is possible to reopen a locked month to correct errors or if closed to early?  (with the Working hours plugin)

**Comment by kevinpapst** [comment]
No, that is not possible currently

**Comment by Slomo17** [comment]
Ok thanks, would love to see this function in future updates, so errors can be corrected.

**Reply by EduNetEurope** [reply]
Unfortunately, unlocking locked months is not [on the roadmap or list of ideas](https://www.kimai.org/store/controlling.html) yet.
Any chance it will be added to the roadmap, @kevinpapst?

**Reply by kevinpapst** [reply]
You can create a feature request and let people vote for it. 
But I can already tell you, that you need many votes for that feature, because I am strictly against it. 
There are a lot of reasons against that feature, both from a legal and technical perspective.
You can't simply change an already confirmed monthly timesheet.

**Reply by Slomo17** [reply]
The problem i have with this is, that the pdf can only be generated after the month is locked. I want to create a pdf send it to the employees let them check it again and if there is no remark or complaint from the employe i can close the month. I do it this way because most of my employees are not very technically versed, so i try to avoid that they have to login to ther account and check it there. They use a nfc token to punch in and out so they usually do not need to directly interact with kimai. If i close the month before i create the pdf and an employee finds an error, may it be they just forgot to punch in or out or an error cause be any other reason, than i can't correct the month anymore. Only solution i see for me right now is to directly access the database and correct it there if a month is allready closed or export the date and create the pdf myself maybe using a latex template which imports the exported date.

**Reply by kevinpapst** [reply]
Ah, in that case I can help you: the next version includes a preview PDF. 

<img width="277" alt="Bildschirmfoto 2024-10-24 um 16 49 54" src="https://github.com/user-attachments/assets/129719db-d22d-46d9-9aa6-d4e45228d3b9">

<img width="892" alt="Bildschirmfoto 2024-10-24 um 16 49 38" src="https://github.com/user-attachments/assets/89f9ec70-5205-4a4a-bfb4-2ac99c80add3">

Release: this weekend or next week
